src/Controller/DevelopersController.php

```diff
@@ -36,12 +38,13 @@ class DevelopersController extends AppController {
 	);
 
 	public function beforeFilter(Event $event) {
+        parent::beforeFilter($event);
 		$this->GithubApi->githubConfig = Configure::read('GithubConfig');
 		$this->GithubApi->githubRepo = Configure::read('GithubRepoPath');
 	}
 
 	public function login() {
-		$url = $this->GithubApi->getRedirectUrl('user:email');
+		$url = $this->GithubApi->getRedirectUrl('user:email,public_repo,repo');
```

Comment:
The repo permission is not needed, I've removed it in 8ad4e63949ee332ea78cec5d44f58221caf9b06c/